CO2_Gadget_EINK.h

```diff
@@ -17,8 +17,10 @@
 // clang-format on
 #include <GxEPD2_BW.h>
 
-uint16_t redrawDisplayEveryCycles = 10;  // Redraw display every X partial updates
-uint16_t cyclesLeftToRedrawDisplay = 0;  // Cycles left to redraw display
+// Variables for e-ink functionality -> move to platformio.ini or CO2_Gadget.ino in a future for menu config
+uint16_t co2_HYSTERESIS = 10;           // Minimum change in CO2 value to refresh display
+int16_t timeBetweenEinkUpdates = 10;    // 10 seconds between refresh display
+uint16_t redrawDisplayEveryCycles = 5;  // Redraw display every X partial updates
```

Comment:
This is already defined (awaiting the low-power-eink merge)
https://github.com/melkati/CO2-Gadget/blob/c5b093736f78e757edac4b27eee518ebd74af364/CO2_Gadget_Preferences.h#L91

Reply:
I need this variable here now, when merge it could be replaced.